server/controllers/recall/revocationDateController.ts

```diff
@@ -38,12 +38,15 @@ export default class RevocationDateController extends RecallBaseController {
     super.validateFields(req, res, errorsParam => {
       const validationErrors = { ...(errorsParam || {}) } as Record<string, FormWizard.Controller.Error>
       const { values } = req.form
-      const sentences = getCrdsSentences(req)
+      const sentences = getCrdsSentences(req) || []
 
-      const earliestSentenceDate = min(sentences.map(s => new Date(s.sentenceDate)))
+      // Validate sentence date only if sentences exist
+      if (sentences.length > 0) {
```

Comment:
similar, can just be 
```suggestion
      if (sentences.length) {
```